src/WebJobs.Extensions.DocumentDB/Trigger/CosmosDBTriggerBinding.cs

```diff
@@ -81,4 +81,4 @@ public ParameterDescriptor ToParameterDescriptor()
             };
         }
     }
-}
+}
```

Comment:
In this class, you can make `DocumentCollectionLocation` and `LeaseCollectionLocation` `internal`, which will give you access to them from the tests. This will let you verify that things were set up as you expected.